sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs

```diff
@@ -1968,18 +1968,18 @@ public virtual Response<DataSourceCredential> CreateDataSourceCredential(DataSou
         }
 
         /// <summary>
-        /// Updates an existing <see cref="DataSourceCredential"/>. In order to update your credential, you cannot create a <see cref="DataSourceCredential"/>
+        /// Updates an existing <see cref="DataSourceCredentialEntity"/>. In order to update your credential, you cannot create a <see cref="DataSourceCredentialEntity"/>
         /// directly from its constructor. You need to obtain an instance via <see cref="GetDataSourceCredentialAsync"/> or another CRUD operation and update
         /// it before calling this method.
         /// </summary>
-        /// <param name="dataSourceCredential">The <see cref="DataSourceCredential"/> model containing the updates to be applied.</param>
+        /// <param name="dataSourceCredential">The <see cref="DataSourceCredentialEntity"/> model containing the updates to be applied.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns>
-        /// A <see cref="Response{T}"/> containing the result of the operation. The result is a <see cref="DataSourceCredential"/>
+        /// A <see cref="Response{T}"/> containing the result of the operation. The result is a <see cref="DataSourceCredentialEntity"/>
         /// instance containing information about the updated credential.
```

Comment:
should this one also finish with entity? (same for other update method)

Reply:
I think it's okay to refer to them only as "credential" given the right context.